data/sql/updates/pending_db_world/rev_1732828124635270700.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Adds NPCs Vol'jin and Belgrom Rockmaul as Quest Starters for Regthar Deathgate
+INSERT INTO `creature_queststarter` (`id`, `quest`) VALUES
```

Comment:
Missing delete before

Reply:
> @sudlud 
> 
> ```sql
> 
> DELETE FROM `creature_queststarter` WHERE (`quest` = 1361); 
> 
> ```
> 
> this?

This would probably delete too much, it should just delete the 2 entries you're then adding in the PR.

Reply:
Not at pc rn, not sure if will, I only need to delete the entries I'm adding right? Because this quest has 1 npc already